modules/openapi-generator/src/main/resources/C-libcurl/any_type-header.mustache

```diff
@@ -0,0 +1,5 @@
+/*
```

Comment:
If you need add header or source files, you can add a "PreTarget.cmake", e.g. https://github.com/kubernetes-client/c/blob/master/kubernetes/PreTarget.cmake

Reply:
The `any_type.h` header was not our invention, the generator sometimes puts `#include "any_type.h"` lines in the code, so this was needed to get it to build. A better solution would be to check why these lines are getting added in the first place, but either way this looks like a bug, not something that should be fixed by the user.

By the way [this](https://github.com/ARM-software/avh-api/tree/master/c) is the api we are working with, in case you want to check why we are running into different problems than you.

Reply:
Thank you for the clarification. I'll go to check it.

Reply:
This header is generated if the api yaml has anyof present other than that of enums. Currently anyof, oneof and allof are not handled. Anyof is handled but only for enums, there is a hack in java code for this.

Reply:
The problem is that the `#include` lines are added, but the header itself is not generated (at least for our api), so the build is broken. I got it to work by adding an empty `any_type.h` header, but maybe you know a better way.